franka_gazebo/config/franka_hw_sim.yaml

```diff
@@ -1,9 +1,9 @@
 arm_id: $(arg arm_id)
 
-m_ee:  0.76
+m_ee: 0.76
 franka_gripper:
-  type:    franka_gazebo/FrankaGripperSim
-  arm_id:  $(arg arm_id)
+  type: franka_gazebo/FrankaGripperSim
+  arm_id: $(arg arm_id)
 
   finger1:
     gains: { p: 100, i: 25, d: 20 }
```

Comment:
Just a quick reminder for ourselves that these PID gains need to be improved before this pull request is merged (see https://github.com/frankaemika/franka_ros/pull/173#issuecomment-938507534 for more information).

Reply:
Based on https://github.com/frankaemika/franka_ros/pull/173#issuecomment-964235123 this commit should be removed.